examples/boost/CMakeLists.txt

```diff
@@ -1,21 +1,39 @@
-cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.14...3.23)
 
 project(CPMExampleBoost)
 
 # ---- Create binary ----
 
 add_executable(CPMExampleBoost main.cpp)
-target_compile_features(CPMExampleBoost PRIVATE cxx_std_17)
+target_compile_features(CPMExampleBoost PRIVATE cxx_std_11)
+
+add_executable(CPMExampleAsio deferred.cpp)
+target_compile_features(CPMExampleAsio PRIVATE cxx_std_17)
```

Comment:
One example source for boost should be enough, this should not demonstrate the library, but rather be a minimal project for use with CPM.

Reply:
OK